### YamlMime:ManagedReference
items:
- uid: System.SystemException
  id: SystemException
  children:
  - System.SystemException.#ctor
  - System.SystemException.#ctor(System.String)
  - System.SystemException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.SystemException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: SystemException
  nameWithType: SystemException
  fullName: System.SystemException
  type: Class
  summary: "Fungerar som basklass för undantag-namnområde."
  remarks: "Den här klassen har angetts som ett sätt att skilja mellan system undantag och undantag för programmet. Det är basklassen för sådana undantag som <xref:System.ArgumentException>, <xref:System.FormatException>, och <xref:System.InvalidOperationException>.</xref:System.InvalidOperationException> </xref:System.FormatException> </xref:System.ArgumentException>      > [!IMPORTANT] > Eftersom SystemException fungerar som basklass för olika typer av undantag, din kod inte utlösa ett undantag för SystemException eller bör det försöker hantera ett SystemException undantag om du inte tänker nytt utlösa det ursprungliga undantaget.       SystemException använder HRESULT COR_E_SYSTEM, som innehåller värdet som 0x80131501."
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public class SystemException : Exception
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.SystemException.#ctor
  id: '#ctor'
  parent: System.SystemException
  langs:
  - csharp
  name: SystemException()
  nameWithType: SystemException.SystemException()
  fullName: System.SystemException.SystemException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Initierar en ny instans av den <xref href=&quot;System.SystemException&quot;> </xref> klass."
  remarks: "Standardkonstruktören <xref:System.Exception.Message%2A>-egenskapen för den nya instansen till en tillhandahålls av systemet meddelande som anger vilket fel som &quot;ett systemfel har uppstått.&quot;</xref:System.Exception.Message%2A> Det här meddelandet tar hänsyn till den aktuella systemkulturen.       I följande tabell visas de inledande egenskapsvärdena för en instans av <xref:System.SystemException>.</xref:System.SystemException>      | Egenskapen | Värdet |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| En null-referens (`Nothing` i Visual Basic). |   | <xref:System.Exception.Message%2A>| Den lokaliserade felsträngen meddelande. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public SystemException ();
    parameters: []
  overload: System.SystemException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.SystemException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.SystemException
  langs:
  - csharp
  name: SystemException(String)
  nameWithType: SystemException.SystemException(String)
  fullName: System.SystemException.SystemException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Initierar en ny instans av den <xref href=&quot;System.SystemException&quot;> </xref> klass med ett angivet meddelande."
  remarks: "Innehållet i den `message` parameter ska vara förstå för användaren. Anroparen av den här konstruktorn är behövs för att säkerställa att den här strängen lokaliserade för den aktuella systemkulturen.       I följande tabell visas de inledande egenskapsvärdena för en instans av <xref:System.SystemException>.</xref:System.SystemException>      | Egenskapen | Värdet |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| En null-referens (`Nothing` i Visual Basic). |   | <xref:System.Exception.Message%2A>| Meddelandet felsträng. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public SystemException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Det meddelande som beskriver felet."
  overload: System.SystemException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.SystemException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.SystemException
  langs:
  - csharp
  name: SystemException(SerializationInfo,StreamingContext)
  nameWithType: SystemException.SystemException(SerializationInfo,StreamingContext)
  fullName: System.SystemException.SystemException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Initierar en ny instans av den <xref href=&quot;System.SystemException&quot;> </xref> klass med serialiserade data."
  remarks: "Den här konstruktorn anropas under deserialisering till att fylla Undantagsobjektet som överförs via en dataström. Mer information finns i [XML och SOAP-serialiseringen](~/add/includes/ajax-current-ext-md.md)."
  syntax:
    content: protected SystemException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Det objekt som innehåller objektdata som serialiserade."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Detaljerad information om käll- och målservrar."
  overload: System.SystemException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.SystemException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.SystemException
  langs:
  - csharp
  name: SystemException(String,Exception)
  nameWithType: SystemException.SystemException(String,Exception)
  fullName: System.SystemException.SystemException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Initierar en ny instans av den <xref href=&quot;System.SystemException&quot;> </xref> klass med ett angivet meddelande och en referens till det interna undantaget som är orsaken till det här undantaget."
  remarks: "Ett undantag som utlöses när ett direkt resultat av ett tidigare undantag kan innehålla en referens till det tidigare undantaget i den <xref:System.Exception.InnerException%2A>egenskapen.</xref:System.Exception.InnerException%2A> Den <xref:System.Exception.InnerException%2A>-egenskap returnerar samma värde som överförs till konstruktorn eller en null-referens (`Nothing` i Visual Basic) om den <xref:System.Exception.InnerException%2A>egenskapen tillhandahåller inte värdet i det interna undantaget för konstruktorn.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       I följande tabell visas de inledande egenskapsvärdena för en instans av <xref:System.SystemException>.</xref:System.SystemException>      | Egenskapen | Värdet |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Referens för inre undantag. |   | <xref:System.Exception.Message%2A>| Meddelandet felsträng. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public SystemException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Felmeddelandet som förklarar orsaken till undantaget."
    - id: innerException
      type: System.Exception
      description: "Undantag som är orsaken till det aktuella undantaget. Om den `innerException` parametern är inte en null-referens (<xref uid=&quot;langword_csharp_Nothing&quot; name=&quot;Nothing&quot; href=&quot;&quot;> </xref> i Visual Basic), det aktuella undantaget utlöses i en <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> block som hanterar det interna undantaget."
  overload: System.SystemException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.SystemException.#ctor
  parent: System.SystemException
  isExternal: false
  name: SystemException()
  nameWithType: SystemException.SystemException()
  fullName: System.SystemException.SystemException()
- uid: System.SystemException.#ctor(System.String)
  parent: System.SystemException
  isExternal: false
  name: SystemException(String)
  nameWithType: SystemException.SystemException(String)
  fullName: System.SystemException.SystemException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.SystemException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.SystemException
  isExternal: false
  name: SystemException(SerializationInfo,StreamingContext)
  nameWithType: SystemException.SystemException(SerializationInfo,StreamingContext)
  fullName: System.SystemException.SystemException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.SystemException.#ctor(System.String,System.Exception)
  parent: System.SystemException
  isExternal: false
  name: SystemException(String,Exception)
  nameWithType: SystemException.SystemException(String,Exception)
  fullName: System.SystemException.SystemException(String,Exception)
- uid: System.SystemException.#ctor*
  parent: System.SystemException
  isExternal: false
  name: SystemException
  nameWithType: SystemException.SystemException
